docs/source/model_parallel.rst

```diff
@@ -0,0 +1,11 @@
+.. role:: hidden
```

Comment:
Is this hidden on purpose? I think it should be fine to not hide this for now since it'll only be in master and won't show up in some stable docs.

Reply:
Should we keep them hidden if the docs aren't polished completely?

Reply:
(Also in response to similar question from @mrshenli) 
As @pritamdamania mentioned, it's fine to have unfinished docs out in master; however, tagging them as hidden can be a helpful way to keep track of what still needs work prior to the final release.